components/Faq.vue

```diff
@@ -1,27 +1,124 @@
 <template>
-  <div class="Parent">
-    <h2 class="Parent-Heading">
-      <!-- Tab TBD -->
-    </h2>
-    <SideNavigationOverview
-      :categories="items"
-      :active-category="activeCategory"
-      @clicked="scrollToCategory"
-    />
-    <!-- Add Scroll Area if SideNavigationOverview is shown -->
-    <div
-      :class="{ 'FaqContent-Scroll-Area': !isSmallScreenWidth }"
-      @scroll.passive="debounce"
-    >
-      <div class="FaqContent-Scroll-Length" :style="{ height: scrollLength }">
-        <div
-          v-for="(item, i) in items"
-          :id="`faq-content-${i}`"
-          :key="i"
-          class="FaqContent-Wrapper"
-        >
-        </div>
+  <div class="Faq">
+    <div class="Faq-Header">Frequently Asked Questions</div>
+    <div class="Faq-Question1">
+      <div class="Faq-Question">Where does all this data come from?</div>
+      <div class="Faq-Expand">+</div>
+    </div>
+    <div class="Faq-Question2">
+      <div class="Faq-Question-Multi">
+        When are upcoming elections?
+        Which races have an impact on police spending, or policing in general?
       </div>
+      <div class="Faq-Expand">+</div>
+    </div>
+    <div class="Faq-Question3">
+      <div class="Faq-Question">Who can I call or write to voice my opinion?</div>
+      <div class="Faq-Expand">+</div>
     </div>
   </div>
 </template>
+
+<style lang="scss" scoped>
+@font-face {
+  font-family: "Avenir";
+  src: url("../assets/AvenirLTStd-Medium.otf");
+}
+.Faq {
+  position: absolute;
+  width: 1440px;
+  height: 497px;
+  left: 0px;
+  top: 936px;
+  background: #f1f8f8;
+  &-Header {
+    position: absolute;
+    width: 553px;
+    height: 33px;
+    left: 443px;
+    top: 64px;
+    font-family: Avenir;
+    font-style: normal;
+    font-weight: 800;
+    font-size: 24px;
+    line-height: 33px;
+    text-align: center;
+    color: #000000;
+  }
+  &-Question1 {
+    position: absolute;
```

Comment:
It would be great if we could share the attributes that all three Questions use in one place. Is it possible to consolidate all 3 into one and use an additional style method to set the unique values

ex:
`&-Question {
    position: absolute;
}
  &-Q1{
     top: 137px;
}
  &-Q2{
     top: 220px;
}
&-Q3{
     top: 303px;
}`

where div names is Faq-Question-Q1 etc.

Reply:
@jwiederhold done!